test/DynamoCoreTests/Core/AssemblyLoadedEventArgsTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Reflection;
+using NUnit.Framework;
+using static Dynamo.Models.NodeModelAssemblyLoader;
+
+namespace Dynamo.Tests.Core
+{
+    [TestFixture]
+    public class AssemblyLoadedEventArgsTests
+    {
+        [Test]
+        [Category("UnitTests")]
+        public void AssemblyLoadedEventArgsConstructorTests()
+        {
+            var assemblyMock = Assembly.GetExecutingAssembly();
+            AssemblyLoadedEventArgs eventArgs = new AssemblyLoadedEventArgs(assemblyMock);
+
+            Assert.AreEqual(assemblyMock, eventArgs.Assembly);
```

Comment:
How about the code coverage under NodeModelAssemblyLoader? Is there a task to cover that or will be included in follow up PRs?

Reply:
NodeModelAssemblyLoader contains the class AssemblyLoadedEventArgs. So this is a small increment in the coverage of the former.

Reply:
@byorda-glb Understood. I mean do you plan to add coverage to `NodeModelAssemblyLoader` later?

Reply:
I don't think so. The code that remains uncovered is composed of events and catch blocks, which are very tricky to test. If we need more coverage in Core, I'd look elsewhere first.

Reply:
@byorda-glb Thank you for clarification. I think this PR is relatively safe to go in. I will merge it as it but @mjkkirschner may contact on fixing some previous tests